### ISSUE TITLE
Jackett doesnt start on armv7 host

### ISSUE THREAD

**Issue by cainmp** [linuxserver/docker-jackett#82]
I created the docker container using the following command:
`docker create --name=jackett -e TZ=America/New_York -p 9117:9117 -v /shares/Docker/jackett:/config --restart unless-stopped linuxserver/jackett`
Everything appears to work correctly, the container starts, but the Jackett app never actually launches.
Log File:
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 10-adduser: executing...
usermod: no changes
[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing...
[cont-init.d] 30-config: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.

Host is a WD MyCloud EX2 Ultra, running Docker 18.09.1

**Comment by thelamer** [comment]
I cannot replicate this, I can tell you that we switched to net-core builds of this from mono a while back, can you see if the `linuxserver/jackett:v0.11.1-ls12` tag works for you ? 

If this works and latest does not then the net-core builds will likely never work on your WD hardware, these are program blobs and outside of our control we just consume them from Jackett build output.

**Comment by cainmp** [comment]
Ok, Thanks for checking. I tried with that version and it fired right up as expected. I knew this older/underpowered hardware may be an issue, but I hoped docker would have helped with that.

I also tried the older lsioarmhf/jackett version prior to your response, and that also launched the jacket app correctly.

Do I have any other options? or do I just need to find a Jackett container that doesn't use net-core?
